### ISSUE TITLE
Revamp entire store prices (including Store UI remake)

### ISSUE THREAD

**Issue by CounterCraftz** [GMDevinity/FloodIssues#438]
I feel like there should be some balances made to the store because majority of the models are in the 40-80 token range 

5 models being 50 tokens, 
5 models being 60 tokens and whatnot

Whilst I understand on how difficult it is to set prices because of the currency we use. I also feel like the price on the models should vary based on how good they look. I'm not sure how to go about but I think models like Daedra should cost a little bit more, similar to dragon slayer and whatnot and of course we need to set expensive prices on some models that should be valued way more than they already look

to recap or something, i feel like the entire store prices should get changed because theres 5 models with the same exact price with barely any extinction at all and to encourage more players to kinda, grind? to get those good models

**Comment by CounterCraftz** [comment]
For posterity:
https://cdn.discordapp.com/attachments/292043580258910209/485892724818444289/unknown.png
